app/src/components/submissionPage/SnapSubmissionStyles.ts

```diff
@@ -0,0 +1,100 @@
+import styled from "styled-components";
+import { MentionsInput } from "react-mentions";
+
+/* A list of styled components for SubmissionTextBox.tsx */
+
+export const ElleImg = styled.img`
+    width: 100%;
+    max-height: 250px;
+`;
+
+export const LeftSideOfScreen = styled.div`
+    height: 430px;
+    background-color: #7040d6;
+    border-radius: 10px 0px 0px 10px;
+    padding: 5%;
+    @media (max-width: 576px) {
+        display: none;
+    }
```

Comment:
While this works, it could also be accomplished by using the bootstrap display utility classes ([docs](https://getbootstrap.com/docs/4.0/utilities/display/)). In this case specifically: `.d-none d-sm-block`.

I would expect that keeping all breakpoint related functionality in bootstrap would make changing the CSS a bit more predictable.

Reply:
For the media query on the RHS element, which requires changing the border radius, I'm wondering if having the LHS using the bootstrap `d-none `d-sm-block` and  the RHS which requires the media query `@media (max-width: 576px)`  would be an issue. I think it might make more sense to extract out a constant used in both to make it more clear? 

Although in the current commit that i have created i have changed the LHS to use the bootstrap utility class.

Reply:
As discussed on Discord, we're going to change the structure slightly so the border radius media query is no longer necessary.